### DISCUSSION TITLE
Translation plan

### DISCUSSION THREAD

**Discussion by noyyyy** [wevm/wagmi#1461] [General]
I found the pr https://github.com/wagmi-dev/wagmi/pull/156 that supports basic i18n but there are not docs provided in other languages. Do the community have a plan to translate document into others language and how do the contributor start it? Btw, I could contribute to Chinese Translation.

**Comment by tmm** [comment]
Anyone can kick off the process by creating a PR! Here's some info on [i18n with Nextra](https://nextra.site/docs/guide/i18n), the docs framework we use.

**Reply by noyyyy** [reply]
Cool! I'll have a try these days.
